expression/builtin_op_vec_test.go

```diff
@@ -64,6 +64,7 @@ var vecBuiltinOpCases = map[string][]vecExprBenchCase{
 	ast.UnaryMinus: {
 		{retEvalType: types.ETReal, childrenTypes: []types.EvalType{types.ETReal}},
 		{retEvalType: types.ETDecimal, childrenTypes: []types.EvalType{types.ETDecimal}},
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETInt}},
```

Comment:
please add testcase to test `mysql.UnsignedFlag` and `uint64(args[i]) > uint64(-math.MinInt64)`

Reply:
@b41sh Thanks, but I looked into `bench_test.go` and it seems our vectorized test suite doesn't support cases that return errors currently. More precisely in `bench_test.go:1036`:
```
err := baseFunc.vecEvalInt(input, output)
c.Assert(err, IsNil, Commentf("func: %v, case: %+v", baseFuncName, testCase))
```
Test will panic if our test cases produce errors.  So I use a range generator to limit the value in `[0, math.MaxInt64]` when unsigned so `uint64(args[i]) > uint64(-math.MinInt64)` will not be satisfied. What's your opinion?
BTW, in `builtin_arithmetic_vec_test.go` these other functions already implemented and tested also carefully avoid such kind of test cases.

Reply:
It's ok, I didn't take that into account.